code/datums/shuttles.dm

```diff
@@ -216,6 +219,8 @@
 	admin_notes = "For player punishment."
 	who_can_purchase = null
 
+//SKYRAT EDIT REMOVAL BEGIN
```

Comment:
This one should stay.

Reply:
i assume you mean the asteroid shuttle
yeah i was a retard for gutting THAT one